internal/cloud/metadata.go

```diff
@@ -89,6 +90,19 @@ func (i *Identifier) identifyGCP() (bool, error) {
 	return regexp.MatchString(".*Google.*", provider)
 }
 
+func (i *Identifier) identifyNutanix() (bool, error) {
+	log.Debug("Checking if the VM is running on Nutanix...")
```

Comment:
I wouldn't assume this is a VM. An output such as "Checking if running under Nutanix" seems more appropriate.

Reply:
Referring not to a `system` 752cf74eb7e2d23bc9e5b6c04e18503c3a3bffc9